jsonpb/jsonpb_test.go

```diff
@@ -462,6 +462,14 @@ func TestMarshaling(t *testing.T) {
 	}
 }
 
+func TestMarshalingNil(t *testing.T) {
+	var msg *pb.Simple
+	m := &Marshaler{}
+	if _, err := m.MarshalToString(msg); err == nil {
+		t.Errorf("mashalling nil returned no error")
```

Comment:
Nit: It seems the convention is usually one l.

s/marshalling/marshaling/

Reply:
Done.